backend/globaleaks/handlers/auth/__init__.py

```diff
@@ -163,6 +190,16 @@ class AuthenticationHandler(BaseHandler):
     """
     check_roles = 'any'
 
+    @staticmethod
+    def cookies_to_dict(cookie_string):
+        cookies = [item.strip() for item in cookie_string.split(';') if item]
+        cookie_dict = dict(item.split('=', 1) for item in cookies)
+        return cookie_dict
+
+    def get_fiscal_code(self):
+        cookies = self.cookies_to_dict(self.request.headers.get(b'cookie', b'').decode())
+        return cookies.get('x-idp-userid')
+
```

Comment:
@joeman65 @alessiofranceschini : GlobaLeaks does not implement cookies for reasons of security design so this code could not be merged: https://docs.globaleaks.org/en/stable/security/ApplicationSecurity.html#cookies-and-xsrf-prevention

To implement such a capability we need to reipmplement this following what we specified in: https://github.com/globaleaks/globaleaks-whistleblowing-software/issues/4107